quickstart/prerequisites.md

```diff
@@ -5,37 +5,208 @@
 All worker nodes must satisfy the following requirements:
 
 * **x86-64** CPU cores with SSE4.2 instruction support:
-  * Intel Nehalem processor \(march=nehalem\) and newer
+  * First generation Intel Core i5 and Core i7 (Nehalem microarchitecture) or newer
   * AMD Bulldozer processor and newer
 
 * Linux kernel **5.4 or higher** with the following modules loaded:
   * nvme-tcp
   * ext4 and optionally xfs
 
-## **Mayastor pods (Data Plane/Engine)**
-Each worker node which will host an instance of a Mayastor pod must have the following resources _free and available_ for use by Mayastor:
+* [Helm](https://helm.sh/docs/intro/install/) version must be v3.7 or later.
 
-* Two CPU cores
-* 4GiB RAM
-* **HugePage support**
-  * A minimum of **2GiB of** **2MiB-sized** pages
+* Supported Kubernetes versions: 
+  - [v1.23.1](https://v1-23.docs.kubernetes.io/)
+  - [v1.22.3](https://v1-22.docs.kubernetes.io/)
+  - [v1.21.8](https://v1-21.docs.kubernetes.io/)
 
-{% hint style="info" %}
-Instances of the Mayastor pod _must_ be run in privileged mode
-{% endhint %}
 
+## Networking requirements
 
-## **Worker Node Count**
+* An Ethernet connection with a minimum speed of 10 Gbps between the nodes must be available at all times.
```

Comment:
That's a best practice, not a prerequisite.  It would run on 1Gbps, if required, with commensurate performance levels.

Reply:
Can we say "recommended" here?

Reply:
Perhaps we should have a Recommended / Best practices page, separate from prerequisites (the latter which should be read as "absolutely essential").

Reply:
Will keep a note. Removing it from here.